commands_test.go

```diff
@@ -1169,7 +1374,7 @@ var _ = Describe("Commands", func() {
 			Expect(val).To(Equal("hello"))
 
 			Eventually(func() error {
-				return client.Get(ctx, "foo").Err()
+				return client.Get(ctx, "key").Err()
```

Comment:
I think it should be "key" and not "foo", since we want to test that "key" expired correctly.
The test passed because "foo" did not exist here, so it was alway nil.

Reply:
Thanks for fixing this :+1: